roles/openshift_health_checker/openshift_checks/logging/fluentd_config.py

```diff
@@ -0,0 +1,144 @@
+"""
+Module for performing checks on a Fluentd logging deployment configuration
+"""
+
+from openshift_checks import get_var, OpenShiftCheckException
+from openshift_checks.logging.logging import LoggingCheck
+from openshift_checks.ovs_version import OvsVersion
+
+
+class FluentdConfig(LoggingCheck):
+    """Module that checks logging configuration of an integrated logging Fluentd deployment"""
+    name = "fluentd_config"
+    tags = ["health"]
+
+    @classmethod
+    def is_active(cls, task_vars):
+        logging_deployed = get_var(task_vars, "openshift_hosted_logging_deploy", default=False)
+
+        # determine OpenShift version
+        try:
+            ovs = OvsVersion(execute_module="")
```

Comment:
Will move `_get_openshift_version` to the parent class.

> Man, this got me really confused... Why do we even have a OvsVersion check and we check the Open vSwitch version using the package_version check?!

The `OvsVersion` check runs as a diagnostic check, while the `package version` Open vSwitch check runs as part of pre-flight checks; they both check for the same thing though